packages/sdks/react-sdk/examples/app/MyApplicationsPortal.tsx

```diff
@@ -0,0 +1,68 @@
+import type { UserResponse } from '@descope/web-js-sdk';
+import React, { useCallback } from 'react';
+import { Link } from 'react-router-dom';
+import { useDescope, useUser, ApplicationsPortal } from '../../src';
+
+const getUserDisplayName = (user?: UserResponse) =>
+  user?.name || user?.loginIds?.[0] || '';
+
+const MyApplicationsPortal = () => {
+  // useUser retrieves the logged in user information
+  const { user } = useUser();
+  // useDescope retrieves Descope SDK for further operations related to authentication
+  // such as logout
+  const sdk = useDescope();
+
+  const onLogout = useCallback(() => {
+    sdk.logout();
+  }, [sdk]);
+
+  return (
+    <>
+      <header
+        style={{
+          borderBottom: '1px solid gray',
+          display: 'flex',
+          justifyContent: 'space-between',
+        }}
+      >
+        <p>
+          <a href="/">Home</a>
+        </p>
+        <div>
+          <p>
+            User:{' '}
+            <span style={{ fontWeight: 'bold' }}>
+              {getUserDisplayName(user)}
+            </span>
+          </p>
+          <p>
+            <button
+              type="button"
+              id="logout-button"
+              onClick={onLogout}
+              style={{
+                display: 'block',
+                marginLeft: 'auto',
+                padding: 5,
+              }}
+            >
+              Logout
+            </button>
```

Comment:
not critical, but I think that our examples should contain only the component, without the user display name and logout and step up, seems like a redundant complexity imo

Reply:
It's like this for all apps so I think it's better to keep consistency
Once we will extract the sample apps into separate packages we can change it